include/deal.II/grid/grid_tools.h

```diff
@@ -1108,6 +1108,11 @@ namespace GridTools
    *   of the cells adjacent to a vertex or an edge/face this function returns.
    *   Consequently, algorithms that call this function need to take into
    *   account that the returned cell will only contain the point approximately.
+   * @param[in] enforce_unique_mapping Enforce one to one mapping between points
+   *   in real and reference space is one to one.
+   * @param[in] marked_vertices Provide vertices that correspond to elements in
+   *   the reference points are searched
```

Comment:
Do you mean
```suggestion
   * @param[in] marked_vertices An array of bools indicating which
   * vertices of @p mesh will be considered within the search
   * as the potentially closest vertex. On receiving a non-empty
   * @p marked_vertices, the function will
   * only search among @p marked_vertices for the closest vertex,
   * otherwise on all vertices in the mesh.
```

Reply:
Thank you, this is much clearer. I will try to be as precise in the future :)